mkdocs.yml

```diff
@@ -66,28 +65,27 @@ nav:
       - 'Change team members roles': 'team/change-team-roles-program.md'
       - 'Remove team members': 'team/remove-team-members-program.md'
     - 'Program Design':
-      - 'Program Design': 'design/design.md'
+      - 'Program Design page': 'design/design.md'
       - 'Read & Change Program Design': 'design/read-change-design-details.md'
     - 'People Affected':
       - 'People Affected page': 'registration/people-affected-page.md'
-      - 'Import Registration excel file': 'registration/registration-import-list.md'
+      - 'Import Registration with excel': 'registration/registration-import-list.md'
       - 'Registration with KOBO form': 'registration/registration-automatic-import-kobo.md'
       - 'Deduplication': 'registration/registration-deduplication.md'
       - 'Edit PA profile': 'registration/edit-pa-profile.md'
       - 'Validate registration': 'registration/registration-validate-pa.md'
-      - 'Reject registration': 'registration/registration-reject-pa.md'
+      - 'Decline registration': 'registration/registration-reject-pa.md'
       - 'Include PA in program': 'registration/inclusion-in-program.md'
       - 'What if I see empty page': 'registration/registration-list-empty.md'
     - 'Payment':
-      - 'Payment Overview': 'payment/payment.md'
-      - 'Update PA payment status': 'payment/change-pa-status-program.md'
-      - 'Import payment instructions in FSP portal': 'payment/issue-payment-fsp-portal.md'
-      - 'Messages in 121 platform': 'payment/list-messages-payments.md'
-      - 'Issue payments': 'payment/issue-payment-integrated-fsp.md'
-      - 'Send and reconcile payment instructions (FSP portal)': 'payment/issue-payment-fsp-portal.md'
-      - 'Notification messages to 121 payment users': 'payment/list-messages-payments.md'
+      - 'Payment page': 'payment/payment.md'
+      - 'Issue instant payments': 'payment/issue-payment-integrated-fsp.md'
+      - 'Send and reconcile manual payment instructions (FSP portal)': 'payment/issue-payment-fsp-portal.md'
+      - 'Pause payments sent to PAs': 'payment/pause-payment-pa.md'
```

Comment:
This looks like you can pause payments that you're sending to PAs, we cannot do that, once a payment is executed you cannot revert it

Reply:
@emily-walkin do you have a way to phrase that better?

Reply:
Pause the inclusion of PAs in payment rounds